src/cli/Clip.cpp

```diff
@@ -90,12 +93,28 @@ int Clip::clipEntry(Database* database, const QString& entryPath, const QString&
         return EXIT_FAILURE;
     }
 
-    int exitCode = Utils::clipText(entry->password());
+    QString value;
+    if (useTotp) {
+        if (!entry->hasTotp()) {
+            err << QObject::tr("Entry with path %1 has no TOTP set up.").arg(entryPath) << endl;
+            return EXIT_FAILURE;
+        }
+
+        value = entry->totp();
+    } else {
+        value = entry->password();
+    }
+
+    int exitCode = Utils::clipText(value);
     if (exitCode != EXIT_SUCCESS) {
         return exitCode;
     }
 
-    outputTextStream << QObject::tr("Entry's password copied to the clipboard!") << endl;
+    if (useTotp) {
+        outputTextStream << QObject::tr("TOTP for entry copied to the clipboard!") << endl;
```

Comment:
@felixfontein I feel like we should preserve the same message format as the one just below.